test/SlowTests/Cluster/CompareExchangeExpirationTest.cs

```diff
@@ -480,6 +481,37 @@ public async Task CanSnapshotCompareExchangeWithExpiration()
             }
         }
 
+        [Fact]
+        public void AllCompareExchangeAndExpirationPreserveAfterServerSchemaUpgradeFrom50000()
+        {
+            var folder = NewDataPath(forceCreateDir: true);
+            DoNotReuseServer();
+
+            var zipPath = new PathSetting("SchemaUpgrade/Issues/SystemVersion/CompareExchange_Expiration_RavenData_v5.2_from50000.zip");
```

Comment:
let's put a issue number in the path e.g.
```
SchemaUpgrade/Issues/SystemVersion/RavenDB_17999/CompareExchange_Expiration_RavenData_v5.2_from50000.zip
```

Reply:
done